pkg/cloud-provider/cloudapi/internal/accounts_common.go

```diff
@@ -272,3 +273,14 @@ func (accCfg *cloudAccountConfig) stopPeriodicInventorySync() {
 		serviceConfig.resetCachedState()
 	}
 }
+
+func (accCfg *cloudAccountConfig) GetVpcInventory() map[string]*runtimev1alpha1.Vpc {
```

Comment:
why is vpcInventory in accounts_common? We need to have separate vpc_common kind of file

Reply:
This file is meant for fetching service configs inside a specific account config. So, it is ok to add code here i guess. 
I think I can get rid of this function altogether and access service configs directly from cloud_common.go.